src/expression_scale_noscale.lua

```diff
@@ -0,0 +1,40 @@
+function plugindef()
+    finaleplugin.RequireSelection = true
+    finaleplugin.Author = "Jacob Winkler"
+--    finaleplugin.AuthorURL = "http://"
+    finaleplugin.Copyright = "CC0 https://creativecommons.org/publicdomain/zero/1.0/"
+    finaleplugin.Version = "v1.0"
+    finaleplugin.Date = "2022/07/30"
+    finaleplugin.AdditionalMenuOptions = [[
+    Expressions: Scaling ON
+    ]]
+    finaleplugin.AdditionalUndoText = [[
+    Expressions: Scaling ON
+    ]]
+    finaleplugin.AdditionalDescriptions = [[
+    Turns on expression scaling in the selected region.
+    ]]
+    finaleplugin.AdditionalPrefixes = [[
+    scale_bool = true
+    ]]
+--    finaleplugin.MinJWLuaVersion = 0.63
+--    finaleplugin.ScriptGroupName = ""
```

Comment:
It would be worth it to add the `finaleplugin.ScriptGroupName` and `finaleplugin.ScriptGroupDescription` to make clearer for users on what the script does on the download page.

https://github.com/finale-lua/lua-scripts/issues/258

These fields directly appear on the website's download page.

<img width="631" alt="Screen Shot 2022-07-30 at 6 05 15 PM" src="https://user-images.githubusercontent.com/5659171/182005301-1d7ae61a-179d-40da-90e0-ae67412585ba.png">